Docs/source/install/hpc/aurora.rst

```diff
@@ -0,0 +1,161 @@
+.. _building-aurora:
+
+Aurora (ALCF)
+==============
+
+The `Aurora cluster <https://docs.alcf.anl.gov/aurora/>`_ is located at ALCF.
+
+
+Introduction
+------------
+
+If you are new to this system, **please see the following resources**:
+
+* `ALCF user guide <https://docs.alcf.anl.gov/>`_
+* Batch system: `PBS <https://docs.alcf.anl.gov/running-jobs/>`_
+* `Filesystems <https://docs.alcf.anl.gov/data-management/filesystem-and-storage/>`_:
```

Comment:
Minor warning suppression: for named links in Sphinx it creates an internal reference and to suppress warnings we can use this to anonymize them (e.g., if the label `Filesystems` is used for more than one link throughout the whole documentation):
```suggestion
* `ALCF user guide <https://docs.alcf.anl.gov/>`__
* Batch system: `PBS <https://docs.alcf.anl.gov/running-jobs/>`__
* `Filesystems <https://docs.alcf.anl.gov/data-management/filesystem-and-storage/>`__:
```